.circleci/config.yml

```diff
@@ -160,6 +160,7 @@ jobs:
     working_directory: /app
     steps:
       - checkout
+      - attach_workspace
```

Comment:
Turns out this was incorrect, but you only find out when CircleCI runs 🤦 

This was the correct invocation: https://github.com/lawrencejones/pgsink/commit/b35a7e732a39af7e269bfa18f9a3d8cd13eeb416